cmd/discovery/cmd/send.go

```diff
@@ -26,7 +26,9 @@ to quickly create a Cobra application.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("send called")
 
-		discovery.SendPing("9001", "9000")
+		fmt.Println(*id)
```

Comment:
seems better to merge line #29 with line #27.

Reply:
they are two lines info